sechub-pds-solutions/tern/docker/Tern-Debian.dockerfile

```diff
@@ -9,19 +9,20 @@ LABEL org.opencontainers.image.title="SecHub Tern+PDS Image"
 LABEL org.opencontainers.image.description="A container which combines Tern with the SecHub Product Delegation Server (PDS)"
 LABEL maintainer="SecHub FOSS Team"
 
-ARG TERN_VERSION="2.12.0"
+ARG TERN_VERSION="2.12.1"
+ARG SCANCODE_VERSION="32.0.4"
 
 # execute commands as root
 USER root
 
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get --quiet update && \
     apt-get --quiet --assume-yes upgrade && \
-    apt-get --quiet --assume-yes install attr bzip2 git jq libgomp1 libpopt0 libxml2-dev libxslt1-dev python3-pip skopeo wget xz-utils zlib1g && \
+    apt-get --quiet --assume-yes install wget attr jq skopeo python3-pip procps tar python3 python3-distutils python-dev git bzip2 xz-utils zlib1g libxml2-dev libxslt1-dev libgomp1 libpopt0 && \
```

Comment:
The original was nicely alphabetically sorted - now it's scrambled and very hard to figure out which packages were added. (apt respects the package dependencies regardless the order of the package names)

Reply:
I will change this.